src/Domain/Infra/Doctrine/MappingConfig.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MsgPhp\Domain\Infra\Doctrine;
+
+/**
+ * @author Pascal Wacker <hello@pascalwacker.ch>
+ *
+ * @internal
```

Comment:
i think we should remove it, as we pass it as a service now

Reply:
the `@internal` annotation?